packages/loader/container-loader/src/container.ts

```diff
@@ -1807,7 +1822,7 @@ export class Container
 		this.setLoaded();
 	}
 
-	private async rehydrateDetachedFromSnapshot(detachedContainerSnapshot: ISummaryTree) {
+	private async rehydrateDetachedFromSnapshotOld(detachedContainerSnapshot: ISummaryTree) {
```

Comment:
pull this out into a free function that just returns the new format. generally, i avoid member functions as they clutter the interface, and are not bundle size efficient as webpack can't shrink their name.